site/sass/components/_item-grid.scss

```
@@ -14,25 +14,50 @@
   position: relative;
   z-index: 10;
 
+  &__title,
+  &__subtitle {
+    text-align: center;
+  }
+
   &__title {
     margin: 0 auto;
     max-width: pxRem(640px);
     padding: 0 calc(var(--shape-circle--size) / 6);
     position: relative;
-    text-align: center;
     z-index: 3;
   }
 
+  &__subtitle {
+    margin: pxRem(8px) auto 0;
```

```suggestion
    margin: .5rem auto 0;
```

A preference to use `rem`s directly wherever possible